transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -127,20 +130,33 @@ private long ensureTransportContext(SQLiteDatabase db, TransportContext transpor
     record.put("backend_name", transportContext.getBackendName());
     record.put("priority", transportContext.getPriority().ordinal());
     record.put("next_request_ms", 0);
+    if (transportContext.getExtras() != null) {
+      record.put("extras", encodeToString(transportContext.getExtras(), DEFAULT));
```

Comment:
Can you switch to `Base64.encodeToString` and `Base64.DEFAULT` for readability? here and throughout